0-bubble_sort.c

```diff
@@ -0,0 +1,42 @@
+#include "sort.h"
+/**
+ * bubble_sort - Sorting, using the bubble sort algorithm
+ * @array: The array to be sorted.
+ * @size: The size of the array  @array
+ *
+ * Return: void.
+ */
+void bubble_sort(int *array, size_t size)
```

Comment:
Alright. Noted.

When I update it, I'll keep you posted.

Reply:
Just curious what happens when an 'empty' array is passed, like I just declared an array, don't initialize any index and pass it to your function, does it work fine?

Reply:
You can't create an array like that in C. C will throw an error that it cannot initialize empty array.

I tried it while testing

Reply:
I have now updated the case, where NULL is passed into the function.

Reply:
Really? that's strange, maybe it's a compiler thing because something like this works in on my computer
```
int arr[50];
```
Though this won't matter as the values in this type of array may vary, so doing a check for it will be tasking and to be sincere it's not worth the effort, you can go ahead and merge now.